client/js/actions/qbank/assessments.js

```diff
@@ -84,6 +85,7 @@ export function createAssessmentOffered(bankId, assessmentId) {
   return {
     bankId,
     assessmentId,
+    body    : { id: '123' }, // have to send up garbage data to make it work.
```

Comment:
does this ID get reset or does it come back as the object id?
If we need to keep this id, I have a guid function in the utils